roles/postgres/tasks/main.yml

```diff
@@ -6,21 +6,29 @@
       - postgresql-contrib-{{ postgresql_version }}
       - libpq-dev
     state: present
+  tags:
+    - role::podman
```

Comment:
All the tags in this file are wrong

Reply:
Hello Joe!

I was heavily honoured to receive a review from this on this matter today. I know that time is short for everyone here, especially you, and I am glad that you managed to grant me three minutes of your free time to review this pull request.

Rest assured that I have addressed your review comment to fullest correctness by running a so-called "search-and-replace" via my text editor "neovim" by using `:%s/podman/postgres/g` in this file, `roles/postgres/tasks/main.yml`. Therefore, I consider this matter closed.